### ISSUE TITLE
[FEA]Is it support BlackWell Architecture

### ISSUE THREAD

**Issue by Emiyagzm** [NVIDIA/cutlass#1975]
I want to know if it support BlackWell architecture, t hanks

**Comment by mnicely** [comment]
Yes, Blackwell will be support in a future release.

**Comment by Emiyagzm** [comment]
Hi  @mnicely ,  thanks for your reply, so Blackwell is not support now?

**Comment by mnicely** [comment]
No. Support for Blackwell, in the NVIDIA SW stack, is not available yet.

**Comment by Emilio66** [comment]
Hi @mnicely, would you please share any plan or roadmap to support Blackwell, when will it be available approximately? 
Since we're working on Blackwell software porting, we relies heavily on CutLASS  which is a great product, highly efficient and reusable. Thanks for your contribution and look forward to your reply :)

**Comment by thakkarV** [comment]
@Emilio66 It will be out after the rest of the CUDA SW stack goes GA

**Comment by Emilio66** [comment]
@thakkarV Thanks for your reply, will the rest of CUDA SW be ready in March 2025 since RTX50xx will go on sale on March as NV mentioned on CES.

**Comment by thakkarV** [comment]
@Emilio66 for future roadmap information, I recommend reaching out to your NVIDIA contacts in private.

**Comment by hwu36** [comment]
blackwell code is out including both sm100 and sm120.
